src/cookie.js

```diff
@@ -12,11 +14,18 @@ function fireSyncs() {
     } else {
       utils.insertPixel(obj.url);
     }
+    setBidderSynced(obj.bidder);
   });
   // empty queue.
   queue.length = 0;
 }
 
+function setBidderSynced(bidder) {
+  setStorageItem(S2S.SYNCED_BIDDERS_KEY, getStorageItem(S2S.SYNCED_BIDDERS_KEY)
+    .concat([bidder])
+    .filter(utils.uniques));
```

Comment:
Why not use an object? `{'appnexus': 1, 'rubicon": 1}`

Uniqueness comes for free, and your lookup in `adaptermanager` should be constant instead of linear.